force-app/main/default/lwc/serviceDeliveryRow/serviceDeliveryRow.js

```diff
@@ -205,12 +232,6 @@ export default class ServiceDeliveryRow extends LightningElement {
         }
     }
 
-    handleServiceInputChange(fieldName) {
```

Comment:
We need to be able to submit with just the service selected.  If you remove contact/PE from the default fieldset, your other fields remain locked - unable to populate.